src/Symfony/Component/Validator/Constraints/Null.php

```diff
@@ -20,8 +20,7 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated
  */
-class Null extends Constraint
-{
-    public $message = 'This value should be null.';
-}
+class_alias('NullConstraint', 'Null');
```

Comment:
In order to keep type hint working (`... instanceof Null`), this class_alias should be moved at the end of NullConstraint.php, and this very line should be replaced by a class_exists('Symfony\Component\Validator\Constraints\NullConstraints')